models/currency.js

```diff
@@ -0,0 +1,188 @@
+const firestore = require('../utils/firestore')
+const walletModel = firestore.collection('wallets')
+const bankModel = firestore.collection('banks')
+const currencyExchangeModel = firestore.collection('currencyExchange')
+const transactionModel = firestore.collection('transactions')
+const { extractReferenceDocumentData, extractReferenceDocumentId } = require('../utils/firestore-helper')
+
+const allCurrencyName = 'allCurrencies'
```

Comment:
Can we have this in constants file? If someone else needs it in future in some other place?